### PR TITLE
Fix europe / fr / Auvergne sources

### PR DESCRIPTION
**Pull request by rbuffat**
The tms sources for Auvergne are not working, but the imagery is also available on the wms server.

This PR:
- removes the not working tms source for "Auvergne 2016 25cm CRAIG"
- switches the "Auvergne 2013 25cm CRAIG" to the wms server 
- Synces the attribution url with josm  https://josm.openstreetmap.de/wiki/Maps/France#Auvergne201625cmCRAIG

### CODE REVIEW

**Review by andrewharvey — APPROVED** [review]
Looks like someone made you Collaborator! Great welcome aboard. I'll try to review your PRs as a check for any comments or anything missed, and approve if okay, will leave to you to merge, just watch out that until travis is fixed, you can't merge two PRs at the same time, need to wait for travis to do its commit first.

**Comment by rbuffat** [comment]
@andrewharvey Thanks. From my side, if you approve one of my PR, you can also directly merge it. I'm not that experienced regarding licenses, so my contributions will probably be more on the technical side. As you probably already noticed, I try currently to fix defunct sources that are already included in the index.

**Pull request merged**
